diff --git a/v4/core/index.js b/v4/core/index.js
--- a/v4/core/index.js
+++ b/v4/core/index.js
@@ -6,7 +6,7 @@
 export * from "./versions.js";
 export * as util from "./util.js";
 export * as regexes from "./regexes.js";
-export * as locales from "../locales/index.js";
+export const locales = {};
 export * from "./registries.js";
 export * from "./doc.js";
 export * from "./api.js";
diff --git a/v4/classic/external.js b/v4/classic/external.js
--- a/v4/classic/external.js
+++ b/v4/classic/external.js
@@ -11,7 +11,7 @@
 export { globalRegistry, registry, config, $output, $input, $brand, clone, regexes, treeifyError, prettifyError, formatError, flattenError, TimePrecision, util, NEVER, } from "../core/index.js";
 export { toJSONSchema } from "../core/json-schema-processors.js";
 export { fromJSONSchema } from "./from-json-schema.js";
-export * as locales from "../locales/index.js";
+export const locales = {};
 // iso
 // must be exported from top-level
 // https://github.com/colinhacks/zod/issues/4491
diff --git a/v4/mini/external.js b/v4/mini/external.js
--- a/v4/mini/external.js
+++ b/v4/mini/external.js
@@ -4,7 +4,7 @@
 export * from "./checks.js";
 export { globalRegistry, registry, config, $output, $input, $brand, clone, regexes, treeifyError, prettifyError, formatError, flattenError, TimePrecision, util, NEVER, } from "../core/index.js";
 export { toJSONSchema } from "../core/json-schema-processors.js";
-export * as locales from "../locales/index.js";
+export const locales = {};
 /** A special constant with type `never` */
 // export const NEVER = {} as never;
 // iso
